models/guilds.js

```diff
@@ -0,0 +1,31 @@
+const Sequialize = require("sequelize");
```

Comment:
are we gonna stick with "Sequialize" or fix that typo now, before doing so will be too much work?